content/blog/entries/2020-03-05-involucrate-gsoc-outreachy-es/contents.lr

```diff
@@ -2,7 +2,9 @@ title: Involúcrate con nuestra comunidad de código abierto a través del  Goog
 ---
 categories:
 gsoc
+gsoc-2019
```

Comment:
these are related to the 2020 round, not the 2019 round, it's to encourage participation

Reply:
Resolved in 223d70e.